### PR TITLE
update to 1.6

### PR DESCRIPTION
**Pull request by notestaff**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by notestaff** [comment]
@jakirkham @mariusvniekerk

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,10 +7,10 @@ package:
 source:
   fn: jq-{{ version }}.tar.gz
   url: https://github.com/stedolan/jq/releases/download/jq-{{ version }}/jq-{{ version }}.tar.gz
-  sha256: c4d2bfec6436341113419debf479d833692cc5cdab7eb0326b5a4d4fbe9f493c
+  sha256: 5de8c8e29aaa3fb9cc6b47bb27299f271354ebb72514e3accadc7d38b5bbaa72
 
 build:
-  number: 1005
+  number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: This is needed for the compiler migration.

```suggestion
  number: 1000
```

**Comment by jakirkham** [comment]
Thanks @notestaff.

One comment above. Also could we bump the `oniguruma` dependency to `6.9.*` (with `6.9.0` used for the build)?

**Comment by notestaff** [comment]
@jakirkham fixed.

**Comment by jakirkham** [comment]
Thanks @notestaff 😄

**Pull request merged**